Parsers/ASimAuditEvent/Tests/Okta_Auth0_ASimAuditEvent_DataTest.csv

```diff
@@ -0,0 +1,15 @@
+﻿Result
+"(0) Error: 1 invalid value(s) (up to 10 listed) in 14 records (100.0%) for field [ActorUsernameType] of type [Enumerated]: [""Oktaid""] (Schema:AuditEvent)"
```

Comment:
ActorUsernameType should be 'UPN'

![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/00a53ba6-77a0-4840-809b-3acee7e6129e)

Reply:
![image](https://github.com/Azure/Azure-Sentinel/assets/97222872/341d27c6-21e5-4f0e-859e-43c6ae1e5b4b)